Mark `normal` and `reset` as non-standard values for `zoom` property (#36732)
mdn/content: files/en-us/web/css/zoom/index.md

## Syntax

```css

> /* Keyword values (Non-standard, use with caution) */
> zoom: normal; /* Non-standard value, not recommended for use */
> zoom: reset; /* Non-standard value, not recommended for use */

Reviewer: We can move this to after the valid values and before the global values.

```suggestion
/* Non-standard keyword values */
zoom: normal; 
zoom: reset;
```

Reviewer: actually, lets just delete the entire section, since "normal" is not listed in the BCD and reset is deprecated.

### Values

- `normal`

>   - : Render this element at its normal size. **Non-standard.** Avoid using this value. Consider using `unset` instead.

Reviewer: ```suggestion
  - : Render this element at its normal size; equal to `zoom: 1`.
```

- `reset`

>   - : Do not (de)magnify this element if the user applies non-pinch-based zooming (e.g. by pressing <kbd>Ctrl</kbd> \- <kbd>-</kbd> or <kbd>Ctrl</kbd> \+ <kbd>+</kbd> keyboard shortcuts) to the document. **Non-standard.** Consider using `unset` instead.

Reviewer: Suggest listing the valid values (percent and number) first, then adding a paragraph that reads something to the effect of:

"Supported keyword values include the following non-standard values:" followed by normal and reset.

>   - : Do not (de)magnify this element if the user applies non-pinch-based zooming (e.g. by pressing <kbd>Ctrl</kbd> \- <kbd>-</kbd> or <kbd>Ctrl</kbd> \+ <kbd>+</kbd> keyboard shortcuts) to the document. **Non-standard.** Consider using `unset` instead.

Reviewer: ```suggestion
  - : Resets the value to `zoom: 1` and prevents the element from being (de)magnified if the user applies non-pinch-based zooming (e.g. by pressing <kbd>Ctrl</kbd> \- <kbd>-</kbd> or <kbd>Ctrl</kbd> \+ <kbd>+</kbd> keyboard shortcuts) to the document.
```

```plain
zoom =

>   normal | reset | <number> | <percentage> /* Non-standard values */

Reviewer: we can get rid of all the text here, and use the macro:

`{{csssyntax}}`

this will then include what is included in the spec (and not the non-standard values)

Author: Thanks for your suggestions, I learn a lot.

## See also

- [`zoom` entry in CSS-Tricks' CSS Almanac](https://css-tricks.com/almanac/properties/z/zoom/)

> - [CSS Unset keyword](/en-US/docs/Web/CSS/unset) - Information on the `unset` keyword, a recommended alternative for resetting properties, such as `zoom`, that are deprecated.

Reviewer: ```suggestion
- {{cssxref("transform")}}
- {{cssxref("scale")}} 
- {{cssxref("unset")}} keyword
- [Legay `zoom` property](https://css-tricks.com/almanac/properties/z/zoom/) via CSS-Tricks (2013)
```
for see also, we generally only include MDN links without additional context and add external links last. With external links, we add the source and year after the link itself. Normally we dont' include 2013 links, but here it's legacy info, so we should likely mention that.
